### PR TITLE
Cooking balance changes

### PR DESCRIPTION
**Pull request by carshalash**
It's mostly just changes to nutriment numbers, gave some buffs to recipes that have harder to get ingredients. Also made it so synthmeat gives less nutriment to actual meat so that there will be some form of reward for raising animals in botany to make food of them.

### CODE REVIEW

**Comment by carshalash** [comment]
Props to my nigga ultraplen.

**Comment by Kurfursten** [comment]
Originally I intended for Primordial Soup to contain mutagen + arithrazine, but at some point I forgot to add it. It's the one that requires 4 mutant plants.

**Comment by carshalash** [comment]
Want me to do it now? Any numbers in specific?

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -1891,7 +1891,7 @@
 
 /obj/item/weapon/reagent_containers/food/snacks/monkeysdelight/New()
 	..()
-	reagents.add_reagent(NUTRIMENT, 10)
+	reagents.add_reagent(NUTRIMENT, 15)
```
Comment (ultraplen) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1894]: This is quite the increase, are monkeys delights that hard to make?

**Comment by kilozombie** [comment]
Rice balls should have more nutriment, they're almost worthless. Rice pudding and boiled rice are also terribly low on nutriment because they have no solid ingredients to add to the pile.
Cinnamon rolls are a ton of effort for (IIRC?) like 6 nutriment, should be buffed a ways.
I think tortilla chips should have 2 nutriment total, so you can actually feed a crew with them as well as a sauce bowl.
Jelly toast, objectively the _tastiest thing_, is very under-filling. Same with the jelly sandwich.

**Comment by ultraplen** [comment]
This PR is barely memeable :^(

**Comment by carshalash** [comment]
I wanted to see more dead monkeys on peoples tables.

**Comment by carshalash** [comment]
Cinammon roll now has as much nutriment as the flan and I buffed honey cinaflan to 1.5 of its original vallue. 

Buffed some rice dishes as well, i'm not really sure about the chip thing though. At the moment don't they have more than 2 nutriment? And they're a bit underutilized already.

**Comment by Kurfursten** [comment]
Whatever amount you settle on, just make sure the chems balance out so that newradiation doesn't literally kill crewmen

**Comment by kilozombie** [comment]
They're eaten in one bite and have 1 nutriment as shown in `food.dm`

**Comment by carshalash** [comment]
OH, you meant the chip basket. Shit I thought you were talking about the individual chip packets.

**Comment by Kurfursten** [comment]
Don't forget you're supposed to dip chips into sauce so that adds to their value

**Comment by carshalash** [comment]
Chawanmushi won't be getting buffed because it's fucking disgusting

**Comment by ghost** [comment]
The sweet sundae ramen should have charcoal

**Comment by Shadowmech88** [comment]
We need a more advanced nutrition system with protein, carbohydrates, and every type of vitamin and mineral.

**Comment by kilozombie** [comment]
kurfursten
KURFURSTEN
I swear if it happens I'll buy kurf like $100 worth of steam games.

**Comment by kilozombie** [comment]
I'll also do at least the bulk of refactoring the individual counts of the reagents for all food

**Comment by Kurfursten** [comment]
I still have the branch open, it's just a lot of boring code that takes a long time in between grad school. There's a lot more mentions of nutrition than you would expect (for instance, when you transform into a simple mob you keep your nutrition). It's still happening though.

**Comment by kilozombie** [comment]
fucking fantastic

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 1 above)
Comment (ghost) [reply code/modules/reagents/reagent_containers/food/snacks.dm:1894]: They're a pest to make for only 10 nutriment and +5 nutriment is nothing. that's not even an additional 4noraisins

Comment (Kurfursten) [reply code/modules/reagents/reagent_containers/food/snacks.dm:1894]: It's actually a pretty decent amount. Monkey's Delight inherits the 10 nutriment from a monkeycube to become 20 nutriment (double the amount of nutriment in the 3 pieces of meat you get from butchering a monkey, which is 9). This buff is raising that to 25.

I'm not sure 4noraisins should be the standard for anything. 6 nutriment is 90 nutrition, e.g.: fairly hearty.

**Comment by ghost** [comment]
@carshalash Edit the chef book to explicitly tell the losers they can just condibottle more nutriment onto their food and that they do in-fact have a reagent scanner they ignore. I don't see them do that enough when they go for rare shit. If you're dedicated to making the rare foods and don't want it gone in 2 guys it's piss easy to get a nutriment-corn oil-sugar farm going and just keep saucing them babies. As long as 2-3 people don't horn dog in on it before you can swoop in you can keep the sprite around all shift.  20+nutriment pumpkins feeding your box of bluespace beakers lets you keep the selection nice with the added benefit of only needing a couple base crops to get a wide selection while the nutriment workhorse is the only thing you really need to fuss with instead of going the splicing route. Much easier imo to just dump a plant bag in the grinder and sauce it up rather than keep remaking heaps of the same food.

**Comment by kilozombie** [comment]
Putting nutriment into food manually is a sin, and I will never do it for any reason.

Also, the kitchen display looks 20x better when there's lots of variety and lots of quantity. If you just have one sandwich with 100 nutriment, expect me to call you out as a shit chef.

also: it's fucking icky when people take 30 bites out of food and then put it down and somebody else takes 30 bites

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 1 above)
Comment (ghost) [reply code/modules/reagents/reagent_containers/food/snacks.dm:1894]: Ah, fair enough.

Raisins are a common reference point that we're all familiar with to put the numbers in scale.

**Comment by ghost** [comment]
I'm saying it's easier to make a wider variety of food and keep that variety with less work.

**Comment by Exxion** [comment]
Just grill something a shit ton of times rather than inject nutriment then

**Comment by kilozombie** [comment]
I also said quantity

**Comment by carshalash** [comment]
Wow that may have just been one of the worst posts I have ever seen and that's coming from an Australian.

**Comment by ghost** [comment]
Think of it as a buffet tray then Kilo

**Comment by kilozombie** [comment]
My customers don't think of it as a buffet tray, they just see one lousy plate
Presentation is the true moniker of a chef, especially when the food in question is expensive (say, it takes a full cheese wheel instead of a wedge)
poutine costed a full cheese wheel at release
Imagine that with only one of everything... that's gonna be one boring display.

**Comment by carshalash** [comment]
How do you deepfry a jar of jam aaaaah

**Comment by kilozombie** [comment]
it's my own recipe

**Comment by Exxion** [comment]
I believe you have fucked up

**Comment by carshalash** [comment]
Oh fuck

**Comment by Intigracy** [comment]
Conflicting files
code/modules/reagents/reagent_containers/food/snacks.dm

**Comment by Intigracy** [comment]
Oh and the superglue change wtf

**Comment by SonixApache** [comment]
@carshalash fix conflicts

**Comment by SonixApache** [comment]
@Intigracy the Australian meme has given up on this PR, might as well close it.

**Comment by kilozombie** [comment]
RIP

**Pull request closed without merging**